src/Compilers/CSharp/Test/Semantic/Semantics/OutVarTests.cs

```diff
@@ -27290,10 +27290,56 @@ public void Test()
             VerifyModelForOutVar(model, x1Decl, x1Ref);
             Assert.Equal("System.String", model.GetTypeInfo(x1Ref).Type.ToTestDisplayString());
         }
+
+        [Fact]
+        public void OutVarDiscard()
```

Comment:
>OutVarDiscard [](start = 20, length = 13)

Need to have tests for more scenarios, indexing, object creation, constructor initializers, etc.